tests/Integration/DashboardLinkTest.php

```diff
@@ -39,7 +39,7 @@ public function set_up(): void {
 	 */
 	public function test_generate_parsely_post_url(): void {
 		$post_id = self::factory()->post->create();
-		$post    = get_post( $post_id );
+		$post    = $this->get_post( $post_id );
```

Comment:
`get_post` returns `WP_Post|null` so we will need a lot of checks in the code where first we will check the `null` and then code the rest. This will not look good in the codebase so to avoid this we implemented `get_post` method in our `TestCase` class which will only return the `WP_Post` to ensures strong typing and cleaner code.

Note: This has been done to some other methods also like
  - get_permalink
  - get_term
  - get_term_link
  
and more.